RELEASING.md

```diff
@@ -5,8 +5,7 @@ Automatic Releasing
  1. Run `bundle exec fastlane bump_and_update_changelog version:X.Y.Z` (where X.Y.Z is the new version) to update the version number in `gradle.properties`, `Config.kt` and in `library/build.gradle`
  1. Commit the changes `git commit -am "Version X.Y.Z"` (where X.Y.Z is the new version)
  1. Make a PR, merge when approved
- 1. Pull main and develop
- 1. Update the main branch. You can do this by pushing develop to main.
+ 1. Pull develop
```

Comment:
i love how all steps are step 1 🤣 i'll update that in my PR though since i'll be making updates for the nexus plugin

Reply:
Oh I actually did that on purpose. When the markdown renders it will number them correctly. I think it's more convenient than having to go an update all numbers manually if one of the items of the list is removed. But it's true that if one of the numbers is not in the correct order like 1, 5, 2, it will still render it fine. So it might make sense to add the correct numbers yes 😆

Reply:
I've always done `1.` for that same reason, but I'm not opposed to actually numbering them